swift/apple/FirezoneKit/Sources/FirezoneKit/Views/MenuBar.swift

```diff
@@ -484,6 +481,19 @@ public final class MenuBar: NSObject, ObservableObject {
     resourceAddressItem.target = self
     subMenu.addItem(resourceAddressItem)
 
+    // Resource toggle
+    if resource.canToggle {
```

Comment:
```suggestion
    if resource.canDisable {
```
If possible

Reply:
need to change connlib for this, can be done but would rather do it in another PR